components/centraldashboard/public/components/main-page.js

```diff
@@ -155,6 +157,7 @@ export class MainPage extends PolymerElement {
      */
     _routePageChanged(newPage) {
         let isIframe = false;
+        this._setNotFoundInIframe(false);
```

Comment:
Try to only propagate the value once, ie. track the value in a variable, and set it once.

Since polymer will instantaneously fire all property handlers as soon as you set the value. Does that make sense?

Reply:
Good point I hadn't really thought much about the effects of setting the property more than once. For some reason I had it in my mind that any change detection was deferred and push to the event loop but I guess I should have read the docs more closely where it states that [Data flow is synchronous](https://polymer-library.polymer-project.org/3.0/docs/devguide/data-system#data-flow-is-synchronous).